modules/diagnostic-bundle-data.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * configuration/generate-diagnostic-bundle.adoc
+:_module-type: CONCEPT
+[id="diagnostic-bundle-data_{context}"]
+= Diagnostic bundle data
+
+When you generate a diagnostic bundle, it includes the following data:
+
+* Central heap profile
+* System logs: Logs of all {product-title} components (for the last 20 minutes) and logs of recently crashed components (from up to 20 minutes before the crash).
+System logs depend on the size of your environment.
+For large deployments, we collect log files for components with critical errors (for example, a high restart count) only.
+* YAML definitions for {product-title} components: This data does not include Kubernetes secrets.
+* {ocp} or Kubernetes events: Details about the events that relate to the objects in the `stackrox` namespace.
+* Online Telemetry data, which includes:
+** Storage information: Details about the database size and the amount of free space available in attached volumes.
+** {product-title} components health information: Details about {product-title} components version, their memory usage, and any reported errors.
```

Comment:
Should version be plural if we're talking about versions for multiple components?